test/ExRam.Gremlinq.Core.Tests/Json/Paths_of_Languages.json

```diff
@@ -0,0 +1,35 @@
+﻿[
```

Comment:
Could you add a test where the GraphSON is version >= 2? There's an example here: http://tinkerpop.apache.org/docs/current/dev/io/#_path_2

Reply:
Added Graphson2 and Graphson3 tests